doc/source/serve/production.md

```diff
@@ -460,9 +460,18 @@ deployment_statuses:
 
 ## Updating Your Serve Application in Production
 
-You can also update your Serve applications once they're in production. You can update the settings in your config file and redeploy it using the `serve deploy` command.
+You can also update your Serve applications once they're in production. You can update the settings in your config file and redeploy it using the `serve deploy` command. You can also add new deployment settings or remove old deployment settings from the config. This is because `serve deploy` is **idempotent**. Your Serve application's config always matches the latest config you deployed successfully– regardless of what config files you deployed before that.
```

Comment:
The first three sentences begin with the same subject "You". Vary it slightly. One way to write this to enumerate actions you can. perform to an existing application in production. One suggestion is:

"A Serve application configuration in production can be modified in a number ways:

- by updating the settings in its config file and redeploying with `serve deploy` CLI
- by adding new or removing old deployment settings from the config file 

All these updates are possible because `serve deploy` is **idempotent**, meaning your Serve application's config always matches (or honors) the latest config in the file.